.github/workflows/ci.yml

```diff
@@ -44,7 +44,9 @@ jobs:
       run : |
         set -euo pipefail
         export GOBIN=$HOME/bin
-        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
+        # TODO(corneliusweig): pin validator version after next release
```

Comment:
what's the reason for this, I'm not sure I get it.

is it because there's no license validator in the tag? if so, can we go-get with a particular commit instead of HEAD?

Reply:
Yes, the license check in the validator was merged after releasing v0.4.0, so we need to use some newer revision.